kbase-extension/kbase_templates/page.html

```diff
@@ -158,5 +158,6 @@
 
     {% block script %}
     {% endblock %}
+    <script src="https://code.jquery.com/jquery-migrate-3.4.0.js"></script>
```

Comment:
is this deliberate if there are no code changes?

Reply:
Ah, crap. Thanks for catching that. Wasn't paying attention to the commit.

Reply:
Fixed, removed that change.